### PR TITLE
Ag 12660/crosshair axis button

### PR DESCRIPTION
**Pull request by manapeirov**
PR for https://ag-grid.atlassian.net/browse/AG-12660

### CODE REVIEW

File: packages/ag-charts-enterprise/src/features/crosshair/crosshair.ts [hunk 1]
```diff
@@ -196,12 +193,12 @@ export class Crosshair extends _ModuleSupport.BaseModuleInstance implements _Mod
...
 
-        if (event.type === 'hover' && this.visible && hoverRect.containsPoint(offsetX, offsetY)) {
-            crosshairGroup.visible = true;
-
+        if (hoverRect.containsPoint(offsetX, offsetY)) {
             const lineData = this.getData(event);
 
             this.updatePositions(lineData);
+
+            crosshairGroup.visible = true;
```
Comment (lsjroberts) [inline packages/ag-charts-enterprise/src/features/crosshair/crosshair.ts:203]: In `hideCrosshairs()` it also hides the labels, do they need to be made visible here too or is that handled elsewhere?

**Review by lsjroberts — APPROVED** [review]
LGTM, one question, but merge when you're happy

File: packages/ag-charts-enterprise/src/features/crosshair/crosshair.ts (hunk 1 above)
Comment (manapeirov) [reply packages/ag-charts-enterprise/src/features/crosshair/crosshair.ts:203]: so that's done in the updatePositions function

**Pull request merged**